src/tests/functional/plugin/gpu/shared_tests_instances/behavior/plugin/auto_batching_tests.cpp

```diff
@@ -1,7 +1,7 @@
 // Copyright (C) 2018-2022 Intel Corporation
 // SPDX-License-Identifier: Apache-2.0
 //
-#include "auto_batching/auto_batching_tests.hpp"
+#include "behavior/plugin/auto_batching_tests.hpp"
```

Comment:
we need the same set of tests for OV 2.0
Can we replicate them? Maybe in a separate PR

CC @myshevts

Reply:
@myshevts Could we delegate this task to Chinese team?